src/readers/morphologyHDF5.cpp

```diff
@@ -213,6 +213,35 @@ void MorphologyHDF5::_readPoints(int firstSectionOffset) {
     auto& somaPoints = _properties._somaLevel._points;
     auto& somaDiameters = _properties._somaLevel._diameters;
 
+    using float_2dvec = std::vector<std::vector<float>>;
+
+    auto loadPoints = [&](const float_2dvec& vec, bool has_neurites) {
+        const std::size_t section_offset = has_neurites
+                                               ? static_cast<std::size_t>(firstSectionOffset)
+                                               : vec.size();
+
+        // points and diameters are TrivialType's. Fastest to resize then assign values
```

Comment:
```suggestion
        // points and diameters are PODs. Fastest to resize then assign values
```
just to get around the apostrophe + plural.